all-together.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<ol><li>Move imports to the top</li><li>Use <code>itertools</code>  to iterate over pairs</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JetBrains-Research/metrics-evaluation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>